examples/src/main/java/io/opencensus/examples/helloworld/QuickStart.java

```diff
@@ -52,12 +52,12 @@
   private static final StatsRecorder statsRecorder = Stats.getStatsRecorder();
   private static final Tracer tracer = Tracing.getTracer();
 
-  // frontendKey allows us to break down the recorded data
+  // frontendKey allows us to break down the recorded data.
   private static final TagKey FRONTEND_KEY = TagKey.create("my.org/keys/frontend");
 
   // videoSize will measure the size of processed videos.
   private static final MeasureLong VIDEO_SIZE = MeasureLong.create(
-      "my.org/measure/video_size", "size of processed videos", "MBy");
+      "my.org/measure/video_size", "size of processed videos", "By");
```

Comment:
Not sure if it's worth changing the video size on L92 or the buckets on L70.

Reply:
I think it makes more sense to change the video size.

Reply:
Changed to MiB range. How does this look?